src/alert.cpp

```diff
@@ -247,14 +248,22 @@ namespace libtorrent {
 		return m_alloc.get().ptr(m_name_idx);
 	}
 
+	char const* file_renamed_alert::old_name() const
+	{
+		return m_alloc.get().ptr(m_old_name_idx);
+	}
+
 	std::string file_renamed_alert::message() const
 	{
 		std::string ret { torrent_alert::message() };
 		char msg[200];
-		std::snprintf(msg, sizeof(msg), ": file %d renamed to "
-			, static_cast<int>(index));
+		std::snprintf(msg, sizeof(msg), ": file %d renamed from \"",
```

Comment:
not an issue by any mean, but are you moving to trailing `,`?

Reply:
unintentional :)

Reply:
😀